__tests__/app.test.js

```diff
@@ -79,6 +79,41 @@ describe("/api", () => {
   });
 });
 
+describe("/api/reviews/:review_id", () => {
+  test("GET - status: 200 - returns an object of review id has corresponding properties", () => {
+    return request(app)
+      .get("/api/reviews/1")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.reviewId[0].hasOwnProperty("review_id")).toBe(true);
+        expect(body.reviewId[0].hasOwnProperty("title")).toBe(true);
+        expect(body.reviewId[0].hasOwnProperty("review_body")).toBe(true);
+        expect(body.reviewId[0].hasOwnProperty("designer")).toBe(true);
+        expect(body.reviewId[0].hasOwnProperty("review_img_url")).toBe(true);
+        expect(body.reviewId[0].hasOwnProperty("votes")).toBe(true);
+        expect(body.reviewId[0].hasOwnProperty("category")).toBe(true);
+        expect(body.reviewId[0].hasOwnProperty("owner")).toBe(true);
+        expect(body.reviewId[0].hasOwnProperty("created_at")).toBe(true);
+      });
+  });
+  test("GET - status: 500 - returns a message when review id is invalid integer", () => {
```

Comment:
Is this a server error (ie the server is broken, the code for it is wrong) or is it a user error? I would say the latter as the user has put in the wrong data type. You handle this in the test below which is great. 

What though if the user made a request with a valid (integer) but it simply did not exist, say for example 99999999 . We need to test for this - have a think about the error code that would make sense in this instance 👍